api-interop-layer/data/alerts/index.test.js

```diff
@@ -73,11 +73,11 @@ describe("alert data module", () => {
               properties: {
                 id: "one",
                 event: "Severe Thunderstorm Warning",
-                sent: new Date().toISOString(),
-                effective: new Date().toISOString(),
-                onset: new Date().toISOString(),
-                expires: new Date().toISOString(),
-                ends: new Date().toISOString(),
+                sent: dayjs().subtract(1, "minute").toISOString(),
+                effective: dayjs().subtract(1, "minute").toISOString(),
+                onset: dayjs().subtract(1, "minute").toISOString(),
+                expires: dayjs().add(1, "minute").toISOString(),
+                ends: dayjs().add(1, "minute").toISOString(),
```

Comment:
After fixing the bug with expired alerts, we also need to be more careful about the test data. Otherwise we get into a race condition where sometimes these tests pass and other times they fail, depending on just how fast the test executes.